src/components/staticCard/index.examples.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
 import {storiesOf} from '@kadira/storybook'
 import {stringFixture, clickHandlerFixture} from '../../utils/Fixtures'
-import DumbComponent from '.'
+import staticCard from '.'
 
-storiesOf('DumbComponent')
+storiesOf('static - Card')
   .addWithInfo('API', () => (
-    <DumbComponent onClick={clickHandlerFixture}>
-      {stringFixture}
-    </DumbComponent>
+    <div>
+      <staticCard onClick={clickHandlerFixture}>
+        {stringFixture}
```

Comment:
Since your `StaticCard` component (with capital `S` 😉 ) won't have `children` or a click handler, I'd suggest writing it as simply `<StaticCard />`